### ISSUE TITLE
Returning null instead of Option

### ISSUE THREAD

**Issue by anze3db** [HairyFotr/linter#2]
This should cause a warning:

```
def test : Option[Type] = null
```

The user probably meant to return **None**.
